modules/rosa-rhoda.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * adding_service_cluster/rosa-available-services.adoc
+:_content-type: CONCEPT
+[id="rosa-rhoda_{context}"]
+= Red Hat OpenShift Database Access
+
+{rhoda} enables easy consumption of database-as-a-service (DBaaS) offerings from partners including MongoDB Atlas, Crunchy Bridge, and CockroachDB directly from managed {product-title} clusters. You can manage, monitor, and create cloud-hosted database instances for connecting to your applications.
```

Comment:
RHODA supports Amazon RDS as well, so shall we update it as "MongoDB Atlas, Crunchy Data Bridge, CockroachDB, or Amazon RDS database instance".

Reply:
@mrrajan Thanks, Rajan, I'll add Amazon RDS to the page.